test_util/test_runner.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+"""Module for running integration_test.py inside of a remote cluster
```

Comment:
It would not be too difficult, but I think we wold need to put a little more effort into how we run tests rather than just plugging in py.test into the installer. I.E. racy tests should have better retry tolerance and the output would need to be restructured.

That being said, I think it would be awesome if postflight was just a thin wrapper for integration_test.py